awscli/examples/directconnect/describe-connections.rst

```diff
@@ -7,15 +7,18 @@ The following ``describe-connections`` command lists all connections in the curr
 Output::
 
   {
-      "connections": [
-          {
-              "ownerAccount": "123456789012", 
-              "connectionId": "dxcon-fg31dyv6", 
-              "connectionState": "requested", 
-              "bandwidth": "1Gbps", 
-              "location": "TIVIT", 
-              "connectionName": "Connection to AWS", 
-              "region": "sa-east-1"
-          }
-      ]
-  }
\ No newline at end of file
+    "connections": [
+        {
+            "awsDevice": "EqDC2-123h49s71dabc", 
+            "ownerAccount": "123456789012", 
+            "connectionId": "dxcon-fguhmqlc", 
+            "lagId": "dxlag-ffrz71kw", 
+            "connectionState": "down", 

```

Comment:
According to the docs its `Down` with a capital.

Reply:
This output is from an actual describe call, so I think the docs probably need to be updated/corrected. I'll make a note to do this.
The reason for this update was because of a request to include a device ID in the example output (which is not returned for a connection that it's in the 'requested' state, as per the previous example). Not all parameters are always returned in a describe call; it depends on the state and configuration of the connection.